pulsar-functions/runtime/src/main/java/org/apache/pulsar/functions/runtime/thread/ThreadRuntime.java

```diff
@@ -206,7 +206,7 @@ public void stop() {
                 // kill the thread
                 fnThread.join(THREAD_SHUTDOWN_TIMEOUT_MILLIS, 0);
                 if (fnThread.isAlive()) {
-                    fnThread.stop();
+                    log.warn("The function instance thread is still alive after {} milliseconds, giving up waiting and moving forward.", THREAD_SHUTDOWN_TIMEOUT_MILLIS);
```

Comment:
Nit. Can we change the message to:

"The function instance thread is still alive after {} milliseconds. Giving up waiting and moving forward to close function."

Reply:
Updated.